packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -557,30 +557,6 @@ const defaultConfigs = {
     minuteLookback: 7200,
     hourlyLookback: 604800,
     minTimeBetweenUpdates: 60
-  },
```

Comment:
why was uSTONKS removed?

Reply:
Oops I'll merge this back in! Great catch